renovate.json5

```diff
@@ -33,8 +32,7 @@
     {
       "description": "LayoutLib shouldn't auto-merge, because updating it is a complex process, see libs/README.md.",
```

Comment:
Does this read me need updating? or the description here?

Reply:
done